### PR TITLE
feat: create component `WorkOrderCell`

### PR DESCRIPTION
**Pull request by Sandtory**
create component `WorkOrderCell`
create `WorkOrderCellScreen`
add navigation for `WorkOrderCellScreen`

### CODE REVIEW

File: apps/chronicles/screens/dfw/dfwcomponents/WorkOrderCellScreen.tsx
Comment (HaakonSvane) [inline apps/chronicles/screens/dfw/dfwcomponents/WorkOrderCellScreen.tsx:1]: I think this page should consist of more example usages of the Cell. You have one example with everything and one with the navigation, but If we add support for switching on and off the symbols, maybe showcase this as well (with some text about it being optional)

**Review by HaakonSvane — APPROVED** [review]
Your code is like the world: full of life, happiness and good things :)

**Comment by Sandtory** [comment]
done deal, boss man 👴🏽

**Review by HaakonSvane — APPROVED** [review]
To my human eyes, the code you have provided exceeds my expectations of what is possible to achieve. This code meets the highest of quality standards and is hereby approved

**Pull request merged**